crates/test_utils/src/connector_auth.rs

```diff
@@ -0,0 +1,257 @@
+use std::{collections::HashMap, env};
+
+use masking::{PeekInterface, Secret};
+use router::types::ConnectorAuthType;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ConnectorAuthentication {
```

Comment:
These structs are not part of test_utils, router applications also uses this struct to read configs. If the router needs this struct then it shouldn't be, we need to place this in a different place.

Reply:
We only use `connector_auth` for testing purposes in the tests folder and no where else. Also, we need the same file to be existent on `test_utils` crate.  
Since we cannot access the file from `tests` folder, I moved that to another crate and accessed the file from there in the `tests` folder.